source/core/utilities.cc

```diff
@@ -485,3 +485,24 @@ get_dimension(const std::string &file_name)
           "Lethe requires that the user explicitly specify the dimension of the problem within the parameter file. This can be achieved by adding set dimension = 2 or set dimension = 3 within the parameter file"));
     }
 }
+
+Tensor<1, 3>
+string_to_tensor3(const std::string &entry_string)
+{
+  std::vector<std::string> vector_of_string(
+    Utilities::split_string_list(entry_string));
+  std::vector<double> vector_of_double =
+    Utilities::string_to_double(vector_of_string);
+
+  if (vector_of_double.size() != 3)
+    {
+      throw(std::runtime_error(
+        "Invalid " + entry_string +
+        ". This should be a three dimensional vector or point."));
+    }
```

Comment:
I would prefer is you used the AssertThrow of deal.II instead of a raw throw like this...
It's more efficient in my perspective.